README.md

```diff
@@ -21,33 +21,31 @@ Jiang, C. and Denolle, M. "NoisePy: a new high-performance python tool for seism
 * adding a jupter notebook for generating response spectrum for a nodal array (to be done)
 
 # Installation
-The nature of NoisePy being composed of python scripts allows flexiable package installation, which is essentially to build dependented libraries the scripts and related functions live upon. We recommand to use [conda](https://docs.conda.io/en/latest/) and [pip](https://pypi.org/project/pip/) to install the library due to their convinence. Below are command lines we have tested that would create a python environment to run NoisePy. Note that the test is performed on `macOS Mojave (10.14.5)`, so it could be slightly different for other OS. 
+The nature of NoisePy being composed of python scripts allows flexible package installation, which is essentially to build dependented libraries the scripts and related functions live upon. We recommend to use [conda](https://docs.conda.io/en/latest/) or [pip](https://pypi.org/project/pip/) to install the library due to their convenience. Below are command lines we have tested that would create a python environment to run NoisePy. Note that the test is performed on `macOS Mojave (10.14.5)`, so it could be slightly different for other OS. 
+
+## Pre-requisites
+
+An MPI installation is required. E.g. for macOS using (brew)[https://brew.sh/] :
+```sh
+brew install open-mpi
+```
 
 ### Note the order of the command lines below matters ###
 
 # With Conda:
-```python
-conda create -n noisepy -c conda-forge python=3.7 numpy=1.16.2 numba pandas pycwt jupyter mpi4py=3.0.1 obspy=1.1 pyasdf
+```bash
+conda create -n noisepy python=3.8 pip
 conda activate noisepy
-git clone https://github.com/mdenolle/NoisePy.git
+pip install -r requirements.txt
 ```
 # With virtual environment:
-```python
+```bash
 python -m venv noisepy
 source noisepy/bin/activate
-pip install wheel h5py numpy numba pandas pycwt jupyter mpi4py pyasdf
-git clone https://github.com/mdenolle/NoisePy.git
+pip install -r requirements.txt
 ```
 To run the code on a single core, open the terminal and activate the noisepy environment before run following command. To run on institutional clusters, see installation notes for individual packages on the module list of the cluster. Examples of installation on Frontera are below.
 
-# Installing on institutional clusters:
```

Comment:
This looks like the same installation steps as the single node installation, so maybe it't not needed?

Reply:
it looks like we wrote this in case some of the packages are already installed in the default environment of some HPC cluster.  PHDF5 is a parallel HDF5 library (https://support.hdfgroup.org/HDF5/PHDF5/). What would you recommend in the case where we would need phdf5 instead pf hdf5 libraries?

Reply:
@mdenolle got it. I missed the first line and the fact that `h5py` is omitted from the module list. , but if someone follows the standard instructions and runs the normal `pip install -r requirements.txt`, would this not work too? Installing hp5y in the virtual environment is not much overhead and we can keep things simple with a single set of instructions.

That said, I'm not familiar with Frontera/phdf5 so if this doesn't work I'm happy to revert. Do you have an environment where this could be tested?

Reply:
OK I asked a colleague at TACC and it still requires h5py so we can. probably ignore that section